index.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const request = require('request');
+const fs = require('fs');
+const Yaml = require('js-yaml');
+
+/**
+ * Publishes the template yaml by posting to the SDAPI /templates endpoint
+ * @method publishTemplate
+ * @return {null}
+ */
+function publishTemplate() {
+    const path = process.env.SD_TEMPLATE_PATH || './sd-template.yaml';
+    const yaml = Yaml.safeLoad(fs.readFileSync(path, 'utf8'));
```

Comment:
`safeLoad` will throw err if the template.yaml is not a valid yaml. Shall we have  some error handling for that? 
https://github.com/nodeca/js-yaml